corral/cmd/_test_helpers.pony

```diff
@@ -0,0 +1,101 @@
+use "files"
```

Comment:
rather than sticking everything in "_test_helpers.pony"

what about a file for the helpers that are only opsrecorder test related?

_test_helpers is very generic and as bad as a "util" package.

so it could be

"_test_data_test_helpers"
"_ops_recorder_test_helpers"

or perhaps better:

_test_helpers_test_data
_test_helpers_ops_recorder

I'm not sure what is best, but I think that we can do better than _test_helpers.

Reply:
I'll break this out in a meaningful way